new_style_grains.md

```diff
@@ -0,0 +1,18 @@
+# New Style Grains
```

Comment:
💯 Nice guide!
Would you mind putting a sentence under the Documentation heading in the README cross-linking to this doc?